.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @tadovas @zolia @vkuznecovas @soffokl
+* @zolia @vkuznecovas @soffokl @waldz @tadaskay @anjmao
```

Comment:
Do we really need codeowners if everyone is code owner? Maybe just write access to the repo is enough?

Reply:
Not everyone, but at this stage, majority of our team should be owners. There is too little of us and we are all experienced enough. Future contributors should not be owners automatically.